src/components/overlays/PendingBadgeOverlay/PendingBadgeOverlay.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useEffect, useRef, useState } from 'react'
+import { Box, CircularProgress, styled, Typography } from '@mui/material'
+import { colors } from '../../../index'
+
+const OverlayContainer = styled(Box)<{ width: number; height: number }>(({ width, height }) => ({
+  width: width,
+  height: height,
+  position: 'absolute',
+  top: 0,
+  left: 0,
+  background: colors.pinkLight,
```

Comment:
I think it will be helpful if we are able to send this background color as an optional property. What do you think?